### ISSUE TITLE
Create modules library/hub

### ISSUE THREAD

**Issue by sakhmedbayev** [sysgears/apollo-universal-starter-kit#406]
Since this awesome kit is using modular structure, I though it would be nice to create a library/hub of modules.

**Comment by vlasenko** [comment]
@sakhmedbayev Yes, we have this in mind. And we are going to add such hub in the future, when we have time to work on it. Help with this is certainly appreciated
